conf/distro/rpb-wayland.conf

```diff
@@ -4,3 +4,7 @@ DISTRO_NAME = "Reference-Platform-Build-Wayland"
 
 DISTRO_FEATURES_append = " wayland"
 DISTRO_FEATURES_remove = "x11"
+
+PREFERRED_PROVIDER_virtual/egl_hikey = "mali450-userland"
```

Comment:
On 06/29/2016 11:43 PM, Nicolas Dechesne wrote:
> In conf/distro/rpb-wayland.conf <https://github.com/96boards/meta-rpb/pull/10#discussion_r69023790>:
>
>> @@ -4,3 +4,7 @@ DISTRO_NAME = "Reference-Platform-Build-Wayland"
>>
>>  DISTRO_FEATURES_append = " wayland"
>>  DISTRO_FEATURES_remove = "x11"
>> +
>> +PREFERRED_PROVIDER_virtual/egl_hikey = "mali450-userland"
>
> why does that need to be here ? it would be more appropriate to be in the machine .conf file. any specific reason?

Just some distros considering this as appropriate for distro config (e.g. [1]).
I had also a version with these PREFERRED_PROVIDER_* statements in hikey.conf, and personally I am OK with the both
implementations.

Thanks,
Andrey

[1] http://arago-project.org/git/?p=meta-arago.git;a=blob;f=meta-arago-distro/conf/distro/include/arago-prefs.inc;h=cc481cb01797bbc4c5ca25e1c87303a8466f85fd;hb=master

> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/96boards/meta-rpb/pull/10/files/e5b7194617d61d814ad21770557697716e88a6fc#r69023790>, or mute the thread
> <https://github.com/notifications/unsubscribe/APSqeR8smsYdwukqa5CRX9r0eOktRHKGks5qQtkJgaJpZM4JBYYO>.
>